salt/modules/mount.py

```diff
@@ -1214,7 +1214,8 @@ def mount(name, device, mkmnt=False, fstype='', opts='defaults', user=None, util
         if fstype:
             args += ' -v {0}'.format(fstype)
     else:
-        args += ' -t {0}'.format(fstype)
+        if fstype:
```

Comment:
@aplanas wouldn't it make sense to refactor a bit here? I can see the `if fstype` being duplicated.

Reply:
Makes sense!

Reply:
done

Reply:
great. thanks!